homeassistant/components/vesync/light.py

```diff
@@ -82,3 +86,36 @@ def supported_features(self):
     def brightness(self):
         """Get dimmer brightness."""
         return round((int(self.dimmer.brightness) / 100) * 255)
+
+
+class VeSyncBulbHA(VeSyncDevice, LightEntity):
```

Comment:
This looks like an exact copy of `VeSyncDimmerHA`. I don't think we need this since the code is the same

Reply:
Maybe make a base class for them to share so we don't have to re-implement all the pieces for both types

Reply:
I thought about the base class too when starting to implement diferentiation for the tunable white temperature version, I'll keep that in mind. 
Also, as I'm diving deeply into more of the code, Maybe I thought some of the names of the variables/objects had some importance. but I think you are right, they are just arbitrary names and nothing else. So I will remove the duplicate class for now, as they are really the same.
Only bad thing is I don't have a wall dimmer switch to test , just the bulbs. so I hope I don't break anything for the devices I don't have.
So, please hold on from merging, I will push another commit first simplifing some stuff and taking your suggestions.

Reply:
Feel free to request a review when its ready as I'll probably miss it otherwise.